rambo/salt_resources/states/hadoop/worker.sls

```diff
@@ -0,0 +1,9 @@
+install_ambari_agent_package:
```

Comment:
This fails with
```
default:           ID: install_ambari_agent_package
default:     Function: pkg.installed
default:       Result: False
default:      Comment: Problem encountered installing package(s). Additional info follows:
default:
default:               errors:
default:                   - Running as unit run-10010.scope.
default:                     E: Unable to locate package ambari-agent
default:      Started: 21:01:16.859841
default:     Duration: 935.619 ms
default:      Changes:
default:
default: Summary for local
default: -------------
```
Which makes sense. There is no apt package called ambari-agent. Did there used to be?
```
vagrant@rambo-joeoffice-test-2d768d8262e4:~$ sudo apt install ambari-agent
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package ambari-agent
```